swagger_server/controllers/operand_controller.py

```diff
@@ -48,4 +68,12 @@ def put_operand(operand_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('id'):
```

Comment:
Not sure we need the ID validation on PUTs. We didn't validate that for GETs and it's pretty similar.